com.unity.perception/Editor/Randomization/Editors/PerceptionEditorAnalytics.cs

```diff
@@ -0,0 +1,99 @@
+﻿using System;
+using JetBrains.Annotations;
+using UnityEditor;
+using UnityEngine.Analytics;
+
+namespace UnityEngine.Perception.Randomization.Editor
+{
+    static class PerceptionEditorAnalytics
+    {
+        static int k_MaxItems = 100;
+        static int k_MaxEventsPerHour = 100;
+        const string k_VendorKey = "unity.perception";
+        const string k_RunInUnitySimulationName = "runinunitysimulation";
+
+        static bool k_IsRegistered = false;
+
+        static bool TryRegisterEvents()
+        {
+            if (k_IsRegistered)
+                return true;
+
+            bool success = true;
+            success &= EditorAnalytics.RegisterEventWithLimit(k_RunInUnitySimulationName, k_MaxEventsPerHour, k_MaxItems,
+                k_VendorKey) == AnalyticsResult.Ok;
+
+            k_IsRegistered = success;
+            return success;
+        }
+
+        enum RunStatus
+        {
+            Started,
+            Failed,
+            Succeeded
+        }
+
+        struct RunInUnitySimulationData
```

Comment:
Might be nice to add the "frames per iteration" value to this list.

Reply:
Hmm, that would be nice but unfortunately would be a big change, since we would also need to update the db. Going to leave this out for now. I hope that at some point we will be able to grab the app params from people's runs directly from USim to get lots of metrics.